src/ralph/cli.py

```diff
@@ -204,6 +206,76 @@ def validate(format_, ignore_errors, fail_on_unknown):
         click.echo(event)
 
 
+@cli.command()
+@optgroup.group("From edX to xAPI converter options")
+@optgroup.option(
+    "-u",
+    "--uuid-namespace",
+    type=str,
+    required=False,
+    default=CONVERTER_EDX_XAPI_UUID_NAMESPACE,
+    help="The UUID namespace to use for the `ID` field generation",
+)
+@optgroup.option(
+    "-p",
+    "--platform-url",
+    type=str,
+    required=False,
+    default="https://fun-mooc.fr",
```

Comment:
Good Idea) Thanks) I was hesitating about making this mandatory because it is specific to edX to xAPI conversion. But until we don't support multiple conversion formats this should be fine. Updated)

Reply:
Maybe click handles conditional requirement? (_i.e._ it should be mandatory when converting from edx format)

Reply:
This would be cool) I will search for this.

Reply:
It's possible to have conditional arguments in click by introducing custom `Option` classes. ([Example](https://stackoverflow.com/questions/44247099/click-command-line-interfaces-make-options-required-if-other-optional-option-is)) I propose to update this part when it will be needed in a future PR)